20_beta_diversity.Rmd

```diff
@@ -135,31 +120,40 @@ p <- p + labs(x = paste("PCoA 1 (", round(100 * rel_eig[[1]], 1), "%", ")", sep
 p
 ```
 
-With additional tools from the ggplot2 package, ordination methods can be 
-compared to find similarities between them or select the most suitable one to
-visualize beta diversity in the light of the research question.
+Multiple ordination plots are combined into a multi-panel plot with the
+patchwork package, so that different methods can be compared to find similarities
+between them or select the most suitable one to visualize beta diversity in the
+light of the research question.
 
 ```{r plot-mds-nmds-comparison, fig.cap = "Comparison of MDS and NMDS plots based on the Bray-Curtis or euclidean distances on the GlobalPattern dataset."}
+# Run MDS on counts assay with Euclidean distances
 tse <- runMDS(tse,
               FUN = vegan::vegdist,
               name = "MDS_euclidean",
               method = "euclidean",
               assay.type = "counts")
 
+# Run NMDS on counts assay with Bray-Curtis distances
 tse <- runNMDS(tse,
                FUN = vegan::vegdist,
                name = "NMDS_BC")
 
```

Comment:
Does this really use bray-curtis or other? It would be good to specify with an argument.